crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -435,24 +435,22 @@ impl Client {
     /// # let client = Client::new(homeserver).await?;
     /// # let room = client.get_room(&room_id!("!test:example.com")).unwrap();
     /// let mut reader = std::io::Cursor::new(b"Hello, world!");
-    /// let encrypted_file = client.upload_encrypted_file(&mime::TEXT_PLAIN, &mut reader).await?;
+    /// let encrypted_file = client.upload_encrypted_file(&mut reader).await?;
     ///
     /// room.send(CustomEventContent { encrypted_file }).await?;
     /// # anyhow::Ok(()) };
     /// ```
     pub fn upload_encrypted_file<'a, R: Read + ?Sized + 'a>(
         &'a self,
-        content_type: &'a mime::Mime,
         reader: &'a mut R,
     ) -> UploadEncryptedFile<'a, R> {
-        UploadEncryptedFile::new(self, content_type, reader)
+        UploadEncryptedFile::new(self, &mime::APPLICATION_OCTET_STREAM, reader)
```

Comment:
I think we can remove this even further, no need for `UploadEncryptedFile` to take this as an argument.